boris-git.toml

```diff
@@ -1,23 +1,11 @@
 [boris]
   version = 1
 
-[build.dist-7-10]
+[build.dist-*]
```

Comment:
@tmcgilchrist Just a suggestion - would suggest doing this on a separate PR, I wasn't sure if the 8.0 build was broken or just a flakey build. I think it's just flakey in this case, but you won't get the branch builds on the PR until this file is updated on `master`.